### ISSUE TITLE
Run XGBoost Model And Check The Results

### ISSUE THREAD

**Issue by SafaNatsheh** [SafaNatsheh/FinalProject#17]
- [x]  Running the model that identifies REM samples against samples from before and after time periods.
- [x] Running the model that identifies and characterizes the N2 period before REM versus general REM.
- [x] Testing if the model detects Overfitting & Under Fitting problems.
- [x] Testing the possibility of improving the model by Cross Validation.
- [x] Check:
               1. [V]  Accuracy
               2. [V]  Recall
               3. [V] Precision
               4. [V] F_Score 
               5. [V] ROC Area Under Curve 
               6. [V] Confusion Matrix
___________________________
